design/common.blocks/button/_theme/button_theme_islands.styl

```diff
@@ -328,7 +328,13 @@ ctx = '.button_theme_islands'
             border-radius: 0;
         }
 
-        &.button_checked,
+        &.button_checked {
```

Comment:
sorry, we don't have code style checks for `.styl` :-/ could you move `{` to next line, plz?